feat: adds capsized typography (#76)
compiled/website: packages/docs/src/pages/what-is-compiled.mdx

# What is Compiled?

> Compiled is a build time [CSS-in-JS](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js) library that has a two-step transformation process that outputs [atomic CSS](/atomic-css).

Reviewer: ```suggestion
library with a two-step transformation process
that outputs [atomic CSS](/atomic-css).
```
avoid the double "that"
